actions/lock-namespace/entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -eu
+
+# use comma as input field separator
+PREVIFS=${IFS}
```

Comment:
Is it important to save and restore `IFS`? I thought this ran in its own environment.

Reply:
I am not sure, although the GitHub runner containers seem to share resources and environment variables, so this is more of a precaution. At minimum this shouldn't hurt.